web/styles/widgets.css

```diff
@@ -215,12 +215,14 @@ button {
         &:focus {
             outline: 0;
             border-color: hsl(156deg 30% 50%);
+            transition: all 0.1s ease;
```

Comment:
We can keep the old transition value too. Also, let's remove the `transition: all 0.2s ease;` so that the bug fix actually works.

Reply:
@amanagr  Should I change the transition time of 0.1 seconds to the old transition duration of 0.2 seconds inside the pseudo selectors?

Reply:
Hi @amanagr ,

I've made the requested changes. The general transition has been removed from the button styles and added to the pseudo selectors :hover, :focus, and :active for the .add-task and .poll-option buttons. This ensures the transition only applies during these states and resolves the issue with the theme switch transition.

Please review the updated code and let me know if further adjustments are needed :)